quickwit/quickwit-doc-mapper/src/query_builder.rs

```diff
@@ -18,57 +18,45 @@
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
 use std::collections::{HashMap, HashSet};
+use std::convert::Infallible;
 
-use anyhow::{bail, Context};
-use quickwit_proto::SearchRequest;
-use tantivy::query::{Query, QueryParser, QueryParserError as TantivyQueryParserError};
-use tantivy::query_grammar::{UserInputAst, UserInputLeaf, UserInputLiteral};
-use tantivy::schema::{Field, FieldEntry, FieldType, Schema};
+use quickwit_query::query_ast::{QueryAst, QueryAstVisitor, RangeQuery};
+use tantivy::query::Query;
```

Comment:
At this point, I wonder if we should systematically rename tantivy imports, at least in this crate and the query crate, with a `Tantivy*` prefix.

Reply:
agreed